internal/src/main/kotlin/com/code42/jenkins/pipelinekt/internal/step/declarative/Mail.kt

```diff
@@ -20,7 +20,17 @@ import com.code42.jenkins.pipelinekt.core.writer.GroovyWriter
  * @param mimeType Email body MIME type. Defaults to text/plain.
  * @param charset Email body character encoding. Defaults to UTF-8.
  */
-data class Mail(val to: Var.Literal.Str = "".strSingle(), val subject: Var.Literal.Str = "".strSingle(), val body: Var.Literal.Str = "".strSingle(), val cc: Var.Literal.Str = "".strSingle(), val bcc: Var.Literal.Str = "".strSingle(), val from: Var.Literal.Str = "".strSingle(), val replyTo: Var.Literal.Str = "".strSingle(), val mimeType: Var.Literal.Str = "".strSingle(), val charset: Var.Literal.Str = "".strSingle()) : DeclarativeStep, SingletonStep {
+data class Mail(
```

Comment:
Moved to multiple lines since detekt was complaining about the line length here.